lib/__testfixtures__/replace-constructors-web.input.js

```diff
@@ -1,2 +1,2 @@
-const de = new DOMException('Error');
+const de = new DOMException("Error");
```

Comment:
Probably best to leave this as-is, so that it's clear that this PR only changes the output of the tests.

Reply:
I was getting an ESLint warning for this

Reply:
We should disable ESLint on `__testfixtures__`

Reply:
Oh that's strange, I did, I'm not sure why it was complaining then